share/default_args.json

```diff
@@ -0,0 +1,17 @@
+{ "--parameters" : {"aliases":["-p"],
+    "type":"str",
+    "help":"Path to the user-defined parameter file",
+    "required":false},
+    "--diags" : {"aliases":["-d"],
+        "type":"str",
+        "nargs":"+",
+        "dest":"other_parameters",
+        "default":[],
+        "help":"Path to other user-defined parameter file",
+        "required":false},
+        "--num_workers": {"aliases":["-n"],
+            "type":"int",
+            "help":"Number of workers, used when runnnig with multiprocessing or distributedly",
+            "required":false},
+            "--scheduler_addr":{"type":"str",
+                "help":"Address of scheduler in the form of IP_ADRESS:PORT. Used when runnnig distributedly"}}
```

Comment:
Might want to fix `runnnig` -> `running`, and probably better "Used when running in distributed mode" - same for above, line 14 comment

Reply:
I'll fix this too. Thanks for catching this stuff.

Reply:
No problem, you probably want to also correct "IP_ADRESS:PORT" -> "IP_ADDRESS:PORT"